nuget/nunit.console-runner.nuspec

```diff
@@ -10,10 +10,18 @@
     <projectUrl>http://nunit.org</projectUrl>
     <iconUrl>http://nunit.org/nuget/nunitv3_32x32.png</iconUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <summary>Console runner for version 3.0 of the NUnit unit-testing framework.</summary>
-    <description>This package includes the nunit3-console runner and test engine for version 3.0 of the NUnit unit-testing framework.
-    
-    The runner will run 2.6 tests using the included plugin.</description>
+    <summary>Console runner for version 3.0 of the NUnit unit-testing framework with common extensions.</summary>
```

Comment:
This looks like the text for NUnit.Console. This shouldn't say that includes the extra extensions.